pkg/components/imguploader/s3uploader.go

```diff
@@ -76,7 +77,7 @@ func (u *S3Uploader) Upload(ctx context.Context, imageDiskPath string) (string,
 	key := u.path + rand + pngExt
 	log.Debugf("Uploading image to s3. bucket = %s, path = %s", u.bucket, key)
 
-	file, err := os.Open(imageDiskPath)
+	file, err := os.Open(filepath.Clean(imageDiskPath))
```

Comment:
After reviewing the gosec [issue thread](https://github.com/securego/gosec/issues/439), I think we should silence this warning, since `filepath.Clean` doesn't resolve any security issue and any security concerns around the folder path should be resolved when originally constructing it.